nix/nomadChart.nix

```diff
@@ -0,0 +1,240 @@
+                    config.ports = ["http"];
+                    user = "0:0";
+                    driver = "docker";
+                    kill_signal = "SIGINT";
+                    kill_timeout = "30s";
+                    resources.cpu = 2000;
+                    resources.memory = 4096;
+                    volume_mount = {
+                      destination = "/persist";
+                      propagation_mode = "private";
+                      volume = "persist-cardano-node-local";
+                    };
+                    vault = {
+                      change_mode = "noop";
+                      env = true;
+                      policies = ["marlowe-runtime"];
+                    };
+                  };
+                  marlowe-history = {
+                    env = {
+                      HOST = "127.0.0.1";
+                      PORT = 9080;
+                      QUERY_PORT = 9081;
+                      SYNC_PORT = 9082;
+                      CHAINSEEKD_HOST = "127.0.0.1";
+                      CHAINSEEKD_PORT = chainseekd.env.PORT;
+                      CHAINSEEKD_QUERY_PORT = chainseekd.env.QUERY_PORT;
+                    };
+
+                    config.image = ociNamer oci-images.marlowe-history;
+                    config.ports = ["http"];
+                    user = "0:0";
+                    driver = "docker";
+                    kill_signal = "SIGINT";
+                    kill_timeout = "30s";
+                    resources.cpu = 2000;
+                    resources.memory = 4096;
+                    vault = {
+                      change_mode = "noop";
+                      env = true;
+                      policies = ["marlowe-runtime"];
+                    };
+                  };
+                  marlowe-discovery = {
+                    env = {
+                      HOST = "127.0.0.1";
+                      PORT = 10081;
+                      QUERY_PORT = 10082;
+                      SYNC_PORT = 10083;
+                      CHAINSEEKD_HOST = "127.0.0.1";
+                      CHAINSEEKD_PORT = chainseekd.env.PORT;
+                      CHAINSEEKD_QUERY_PORT = chainseekd.env.QUERY_PORT;
+                    };
+
+                    config.image = ociNamer oci-images.marlowe-discovery;
+                    config.ports = ["http"];
+                    user = "0:0";
+                    driver = "docker";
+                    kill_signal = "SIGINT";
+                    kill_timeout = "30s";
+                    resources.cpu = 2000;
+                    resources.memory = 4096;
+                    vault = {
+                      change_mode = "noop";
+                      env = true;
+                      policies = ["marlowe-runtime"];
+                    };
+                  };
+                  marlowe-tx = {
+                    env = {
+                      HOST = "127.0.0.1";
+                      PORT = 11080;
+                      CHAINSEEKD_HOST = "127.0.0.1";
```

Comment:
So, these all run on the same node? That will work for now, but that won't be how stuff will scale in the future. We can cross that bridge when we get to it I suppose.